x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/action/ExecuteEnrichPolicyStatus.java

```diff
@@ -22,20 +23,30 @@ private PolicyPhases() {}
         public static final String RUNNING = "RUNNING";
         public static final String COMPLETE = "COMPLETE";
         public static final String FAILED = "FAILED";
+        public static final String CANCELLED = "CANCELLED";
     }
 
     public static final String NAME = "enrich-policy-execution";
 
     private static final String PHASE_FIELD = "phase";
+    private static final String STEP_FIELD = "step";
 
     private final String phase;
+    private final String step;
```

Comment:
Maybe a different name?

Reply:
It's a weird situation because the phases here are more like a "status" than a "phase", but naming things is the worst. Step is a fine name in my opinion